cloud.foundry.cli/src/main/java/cloud/foundry/cli/logic/diff/change/parsing/ContainerChangeParsingStrategy.java

```diff
@@ -35,25 +36,33 @@
     protected List<CfChange> doParse(Change change) {
         ContainerChange containerChange = (ContainerChange) change;
 
+        log.verbose("Parsing change type", change.getClass().getSimpleName(), "to custom change type",
+                CfContainerChange.class.getSimpleName());
         List<CfContainerValueChanged> cfChanges = containerChange.getChanges()
                 .stream()
-                .map(this::parseListEntry)
+                .map(elementChange -> parseListEntry(((ContainerChange) change).getPropertyName(), elementChange))
                 .filter(Objects::nonNull)
                 .collect(Collectors.toList());
 
-        return Arrays.asList(new CfContainerChange(change.getAffectedObject().get(),
+        log.debug("Parsing change type", change.getClass().getSimpleName(), "to custom change type",
+                CfContainerChange.class.getSimpleName(), "completed");
+        return Collections.singletonList(new CfContainerChange(change.getAffectedObject().get(),
                 containerChange.getPropertyName(),
                 extractPathFrom(change),
                 cfChanges));
     }
 
-    private CfContainerValueChanged parseListEntry(ContainerElementChange elementChange) {
+    private CfContainerValueChanged parseListEntry(String propertyName, ContainerElementChange elementChange) {
         if (elementChange instanceof ValueAdded) {
-            return new CfContainerValueChanged(((ValueAdded) elementChange).getAddedValue().toString(),
-                    ChangeType.ADDED);
+            String value = ((ValueAdded) elementChange).getAddedValue().toString();
+
+            log.verbose("Appending", propertyName, "container change with added entry:", value);
```

Comment:
I would recommend to use `debug` logging here to prevent long and confusing output